### PR TITLE
User story 1 worksession

### PR DESCRIPTION
**Pull request by dkannenberg**
Got this one to finally work.  Took me a while to get my rust off, but the exercise was a good one.  

Did the following:

1. Got the timer working to count down
2. Created a "stop" button that stops the timer.  User can continue by "unstopping" the timer if they wanted.
3. Created a reset button.

I think I could also refactor this to create an "Initial" setting, but right now, I hard-coded it all in.

### CODE REVIEW

File: app/pages/index.html [hunk 1]
```diff
@@ -1,11 +1,26 @@
...
-<html>
+<html ng-app="bloctime">
 <head lang="en">
     <meta charset="UTF-8">
     <title>BlocTime</title>
-    <link rel="stylesheet" href="/styles/style.css">
+	<link rel="stylesheet" type="text/css" href="http://fonts.googleapis.com/css?family=Open+Sans:400,800,600,700,300">
+     <link rel="stylesheet" type="text/css" href="http://code.ionicframework.com/ionicons/2.0.1/css/ionicons.min.css">
+    <link rel="stylesheet" href="/styles/bootstrap.css">
+	<link rel="stylesheet" href="/styles/style.css">
```
Comment (RyanBalfanz) [inline app/pages/index.html]: A small thing but the indenting is inconsistent here, and elsewhere in the file. It makes it harder than necessary to read, IMHO.

File: app/scripts/filters/TimeConvert.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+(function() {
+     function timeConvert() {
```
Comment (RyanBalfanz) [inline app/scripts/filters/TimeConvert.js]: Did you look at [Moment.js](http://momentjs.com/) by chance?

File: app/scripts/services/ButtonControl.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+(function() {
+     function ButtonControl() {
```
Comment (RyanBalfanz) [inline app/scripts/services/ButtonControl.js]: @dkannenberg What does this do?

File: app/scripts/services/TimeControl.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
+(function() {
+     function TimeControl($interval) {
```
Comment (RyanBalfanz) [inline app/scripts/services/TimeControl.js]: It seems this is still not being used. Do you plan to use it later?

File: app/scripts/filters/TimeConvert.js (hunk 2 above)
Comment (dkannenberg) [inline app/scripts/filters/TimeConvert.js]: Will be working on this before Wed!

**Pull request merged**